mne/tests/test_evoked.py

```diff
@@ -8,6 +8,7 @@
 from copy import deepcopy
 import os.path as op
 import pickle
+import mne
```

Comment:
we don't usually import the whole `mne` module in our testing files, we just import the functions we need (see lines 19-27)